README.md

```diff
@@ -1 +1,47 @@
-# silverstripe-tx-translator
+# silverstripe tx translator
+
+Module to assist with transifex integration. Works with Silverstripe 4.12+ and 5.0+.
+
+## Operations
+
+- Pull latest translations from transifex and merge them into yml/json/js translation files
+- Run i18nTextCollectorTask on a local website
+- Push updated source strings to transifex (optional)
+- Create github pull-requests with file changes
+
+## Requirements
+
+Transifex cli client (tx) - [https://developers.transifex.com/docs/cli](https://developers.transifex.com/docs/cli) configured API key from transifex.
+
+You must use the go version at least version 1.6+ and not the old python version.
+
+Create a new classic github api token in [github token settings](https://github.com/settings/tokens) with all checkboxes unticked except for `public_repo`. This is required to create pull-requests.
+
+Delete this token from github once you have completed updating translations.
+
+## Usage
+
+```
+cd /path/to/my-local-site
+composer install silverstripe/tx-translator
```

Comment:
```suggestion
composer require silverstripe/tx-translator
```
Sorry, I should have caught this earlier. This is the last thing, then it's all good for merge.

Reply:
Updated